Lib/test/test_capi.py

```diff
@@ -1068,6 +1068,18 @@ def test_state_access(self):
                 with self.assertRaises(TypeError):
                     increment_count(1, 2, 3)
 
+    def test_get_module_bad_def(self):
```

Comment:
You may mention "bpo-46433" in a comment, here and in test_get_module_static_in_mro().